tools/testdata/check_format/cpp_std.cc

```diff
@@ -0,0 +1,11 @@
+#include <memory>
+
+#include "absl/memory/memory.h"
```

Comment:
btw this file (and the gold) is not actually compiled, so you don't need the includes. You do need the 'namespace Envoy' as the presence of that is one of the formatting rules that is checked.

Reply:
Cleaning those out is totally optional IMO.